eos/b-decays/b-to-d-l-nu_TEST.cc

```diff
@@ -140,30 +94,41 @@ class BToDLeptonNeutrinoTest :
                 p3["mu"]                    =  4.18;
                 p3["mass::b(MSbar)"]        =  4.18;
                 p3["mass::c"]               =  1.275;
-                p3["b->cmunumu::Re{cVL}"]   =  1.0;
-                p3["b->cmunumu::Im{cVL}"]   = -1.0;
-                p3["b->cmunumu::Re{cVR}"]   =  2.0;
-                p3["b->cmunumu::Im{cVR}"]   = -2.0;
-                p3["b->cmunumu::Re{cSL}"]   =  3.0;
-                p3["b->cmunumu::Im{cSL}"]   = -3.0;
-                p3["b->cmunumu::Re{cSR}"]   =  4.0;
-                p3["b->cmunumu::Im{cSR}"]   = -4.0;
-                p3["b->cmunumu::Re{cT}"]    =  5.0;
-                p3["b->cmunumu::Im{cT}"]    = -5.0;
+                // mu mode
+                p3["b->cmunumu::Re{cVL}"]         = +1.0;
+                p3["b->cmunumu::Im{cVL}"]         = -2.0;
+                p3["b->cmunumu::Re{cVR}"]         = +2.0;
+                p3["b->cmunumu::Im{cVR}"]         = -2.0;
+                p3["b->cmunumu::Re{cSL}"]         = +3.0;
+                p3["b->cmunumu::Im{cSL}"]         = -3.0;
+                p3["b->cmunumu::Re{cSR}"]         = +4.0;
+                p3["b->cmunumu::Im{cSR}"]         = -4.0;
+                p3["b->cmunumu::Re{cT}"]          = +5.0;
+                p3["b->cmunumu::Im{cT}"]          = -5.0;
+                // tau mode
+                p3["b->ctaunutau::Re{cVL}"]       = +1.0;
+                p3["b->ctaunutau::Im{cVL}"]       = -5.0;
+                p3["b->ctaunutau::Re{cVR}"]       = +2.1;
+                p3["b->ctaunutau::Im{cVR}"]       = -6.0;
+                p3["b->ctaunutau::Re{cSL}"]       = +3.1;
+                p3["b->ctaunutau::Im{cSL}"]       = -7.0;
+                p3["b->ctaunutau::Re{cSR}"]       = +4.1;
+                p3["b->ctaunutau::Im{cSR}"]       = -8.0;
+                p3["b->ctaunutau::Re{cT}"]        = +5.1;
+                p3["b->ctaunutau::Im{cT}"]        = -9.0;
 
                 Options oo;
                 oo.set("model", "WilsonScan");
                 oo.set("form-factors", "BCL2008");
 
                 BToDLeptonNeutrino d(p3, oo);
 
-                const double eps = 1e-5;
+                const double eps = 1e-3;
 
-                // test for different s-bins - the default lepton is muon
-                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(4.0, 8.0), 1074.2803668903907, eps);
-                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(8.0, 11.62), 706.6625032909665, eps);
-                // the full phase-space region for muon
-                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(0.011164, 11.62), 2461.2412433585673, eps);
+                // the default lepton is muon
+                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(0.011164, 11.62), 2581.58, eps);
```

Comment:
I dont understand how the normalized integrated BR can be so large?

   |V_cb|^2 ~ (0.042)^2 ~ 0.00176

then the normal BR

  BR = 0.00176 * 2581.58 ~ 4.55 

becomes larger than 1. How can this be? Did I missunderstand something?

Reply:
@cBobeth This is a NP test case, which has the Wilson coefficients set to unphysically large values.

Reply:
Indeed, if NP were so large, then the lifetime would be modified for sure in some way, such that BR will be smaller than 1 again, if one takes this into account.